source/blog/2017-04-27-rspec-3-6-has-been-released.md

```diff
@@ -0,0 +1,118 @@
+---
+title: RSpec 3.6 has been released!
+author: Sam Phippen, TODO: other editors
+---
+
+RSpec 3.6 has just been released! Given our commitment to
+[semantic versioning](http://semver.org/), this should be an easy
+upgrade for anyone already using RSpec 3, but if we did introduce
+any regressions, please let us know, and we'll get a patch release
+out with a fix ASAP.
+
+RSpec continues to be a community-driven project with contributors
+from all over the world. This release includes over XXX commits and YYY
+merged pull requests from over 50 different contributors!
```

Comment:
It's funny that you have the contributor count filled in but not the commits and PRs!  Usually I fill this in after generating the stats as it is based on that.